### DISCUSSION TITLE
Remnant Defense Mission

### DISCUSSION THREAD

**Discussion by SultanRTG** [endless-sky/endless-sky#9307] [Q&A] [answered]
![image](https://github.com/endless-sky/endless-sky/assets/104505171/40b1865b-92b7-4a9a-8af5-679dc38e41fc)
The wiki says that for the first remnant combat mission, there is a Korath World Ship. Is this outdated (I don't remember fighting a world ship in remnant space before, for me it just skipped to the Pavralet)?

**Comment by warp-core** [comment, marked as answer]
That is indeed outdated. When that mission was first added, the bounty was indeed a World-Ship, but it was later changed to a Korath Dredger (now known as a Rano'erek.)

**Reply by SultanRTG** [reply]
Alright, I was worried I might have missed something because "mission" ships are trivially easy to capture.

**Reply by SultanRTG** [reply]
I'm also stuck on the Remnant Questline again. I got to the point where one of the prefects would get back to me in a few months because of the Patir system (Black hole, with the Builder Settlement containing iridium). I started hitting spaceports and got nothing. Was there a specific timeframe for it?

**Reply by warp-core** [reply]
I think the continuation of that story has yet to be written, or at least, it has not been submitted so far.
